Phases/README.md

```diff
@@ -0,0 +1,37 @@
+# Phases
+
+With the changes introduced in "version 1.5" of the guild, there are now 5 sequential phases that learners move through.
+
+1. [Foundations 1](#foundations-1)
+1. [Foundations 2](#foundations-2)
+1. [Integrated Practice Curriculum (IPC)](#integrated-practice-curriculum-ipc)
```

Comment:
consider renaming this to "Integrated Practice" we dont need anymore initialisms.